packages/components/select/src/useSelect.ts

```diff
@@ -752,6 +752,28 @@ export const useSelect = (props: ISelectProps, emit) => {
     return Number.parseFloat(style.gap || '6px')
   }
 
+  const handleKeydown = (event: KeyboardEvent) => {
+    switch (event.key) {
+      case 'ArrowDown':
+        navigateOptions('next')
+        break
+      case 'ArrowUp':
+        navigateOptions('prev')
+        break
+      case 'Escape':
+        handleEsc()
+        break
+      case 'Enter':
+        selectOption()
+        break
+      case 'Backspace':
```

Comment:
`delete` seems to match both keys. [link](https://vuejs.org/guide/essentials/event-handling.html#event-modifiers#:~:text=.delete%20(captures%20both%20%22Delete%22%20and%20%22Backspace%22%20keys))

Reply:
https://github.com/element-plus/element-plus/pull/12200